python/create_account_with_captcha.py

```diff
@@ -18,8 +18,11 @@
 from flask import Flask, render_template, flash, request
 
 S = requests.Session()
-WIKI_URL = "https://test.wikipedia.org"
-API_ENDPOINT = WIKI_URL + "/w/api.php"
+TEST_WIKI_URL = "https://test.wikipedia.org"
```

Comment:
We are declaring endpoints twice now. You can keep this as it was initially `WIKI_URL` & `API_Endpoint.`

Reply:
Actually, those were meant to be two different endpoints.